### PR TITLE
FAQ Updates

### LINKED ISSUE #82: Update FAQ
Update in faq needed to add couple of questions clearing out the confusion related to tickets.

### PR DESCRIPTION
**Pull request by realslimshanky**
this resolves #82

### CODE REVIEW

File: faqs.html [hunk 1]
```diff
@@ -182,12 +182,28 @@
 												<div class="tg-description">
 													<p><span>A.</span>- Full conference pass for two days.<br>
 													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- Breakfast, Lunch and Hi-Tea.<br>
-													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- PyCon India T-shirt which will be distributed on Sep 25.
+													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- PyCon India T-shirt which will be distributed on November 5th.
```
Comment (CuriousLearner) [inline faqs.html]: Are we sure on this? I think this needs discussion.

**Review by CuriousLearner — COMMENTED** [review]
Rest looks good.

@akshayaurora  What is more needed here?

File: faqs.html (hunk 1 above)
Comment (stalwart201) [reply faqs.html]: This is good @CuriousLearner We can make a change later if required, for now its good to go.

Comment (realslimshanky) [reply faqs.html]: If we've not decided a date yet we can simply remove it.

Comment (CuriousLearner) [reply faqs.html]: Anything works! We should get rid of 25th Sep date right now which is entirely wrong.

**Pull request closed without merging**